manager/testdata/new-blueprints/write-module.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: arrow-flight-common-write-conf
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+    name: arrow-flight-common-write-conf
+spec:
+  description: "This module knows how to configure a cluster wide arrow flight service"
+  # this field is true if this module serves multiple users. Otherwise it
+  # is false. It is used for example when setting the endpoint to
+  # access the module service - if it is a multi-user module then
+  # the service name will be fixed and will be taken from
+  # M4DModule.Capabilities.Api.endpoint.host.
+  multi-user: true
```

Comment:
does it makes sense to rename multi-user field to configuration-module as this field is true if this module configures another service.

Reply:
I don't even think that we need this field at all. As @ronenkat also mentioned what would a controller do with this information? In the end it does not matter what a module deploys. It just has to make sure that it can manage the status and knows how to access the results. No matter if it's configuring a service or starting a K8s job/deployment.

Reply:
I think that it matters for the api.endpoint - if its a configuration module this field must be filled and will hold the common service endpoint otherwise it is like the current implementation- the controller generates the release name based on the steps etc to make sure there are no conflicts in the resources names.

Reply:
Yes. But then the `api.endpoint` is the differentiating factor. For modules that configure a service it's set for others it's not. As we don't have a controller or webhooks we won't check if the module definition is valid anyways.